radar/platforms-and-services/azure-devops.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Azure DevOps"
+ring: hold
+quadrant: "platforms-and-aoe-services"
+tags: [cloud, pipelines, cicd]
+---
+
+[Azure DevOps](https://azure.microsoft.com/it-it/services/devops/) is a set of
+services offered by Microsoft for the development of software in the cloud. It
+is a complete solution that covers the entire software development lifecycle and
+allows you to plan, develop, test, deploy and monitor applications with a single
+tool.
+
+Since we switched to GitHub Actions, that offers a similar set of features, but
+a better integration with GitHub, we are not using Azure DevOps anymore.
+
+Use Github Actions instead, when starting a new project.
```

Comment:
While we do prefer GitHub Actions, we still need Azure DevOps for scheduled pipelines because GitHub shuts down Actions in inactive repos (60d)

Reply:
This is a good point. We should explore if there are other use cases.